### ISSUE TITLE
conditions question

### ISSUE THREAD

**Issue by bkmartinjr** [theislab/scarches#10]
It appears that the models can only be conditioned on a single dataframe column, so if I want to have multiple conditions, I have to create a single column to represent these multiple conditions with unique values.  Is that reading correct or is there a way to condition on multiple columns?

**Comment by M0hammadL** [comment]
Yes, at the moment we are using only single data frame column for conditions. if you want you to have multiple columns, you need have on column that includes all unique conditions for your data points. Imagine you have on column called `tech` that has sequencing technology for each sample, sth like, [smart-seq, 10x,...., smart-seq] if you pass `tech` unique condition and `tech` as `condition_key` to the model then only variations wrt to tech will be removed. If you have another column called `age` which has age of each sample [1, 2, ..., 5] and you also want to remove variation wrt age and tech you need to make a third column e.g "tech_age" = [smart-seq_1, 10x_2, ......] and then pass `tech_age` as condition_key and unique values of this column  to the model to remove both age and tech variation batch effects from the data
